app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
+  &.margin {
+    margin: 0 !important;
+  }
+
+  &.bottom.margin {
+    margin-bottom: 0 !important;
+  }
+
+  &.top.margin {
+    margin-top: 0 !important;
+  }
 }
 
 .less {
   &.padding {
     padding: 0 !important;
   }
-  
-  &.padding-right {
+
+  &.right.padding {
```

Comment:
@harshitagupta30 
```
 &.margin:not(.bottom):not(.top) {
    margin: 0 !important;
  }
```
this will ensure that styling is applied only if the class name does not contain bottom or top along with it, i.e. If it is only `less margin` then only `margin: 0` will be applied. If it were `less top margin` it will not be applied and rather `margin-top: 0` will be applied.